src/physics/em/detail/SeltzerBergerInteractor.hh

```diff
@@ -0,0 +1,85 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file SeltzerBergerInteractor.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "base/Macros.hh"
+#include "base/Types.hh"
+#include "physics/base/CutoffView.hh"
+#include "physics/base/Interaction.hh"
+#include "physics/base/ParticleTrackView.hh"
+#include "physics/base/Secondary.hh"
+#include "base/StackAllocator.hh"
+#include "physics/base/Units.hh"
+#include "physics/material/ElementView.hh"
+#include "SeltzerBerger.hh"
+#include "SBEnergyDistribution.hh"
+#include "SBPositronXsCorrector.hh"
+
+namespace celeritas
+{
+namespace detail
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Seltzer-Berger model for ...
+ *
```

Comment:
As discussed on slack, extend documentation with the algorithms used to sample the interaction.